Lib/test/test_sysconfig.py

```diff
@@ -423,6 +429,8 @@ def test_library(self):
             self.assertTrue(library.startswith(f'python{major}{minor}'))
             self.assertTrue(library.endswith('.dll'))
             self.assertEqual(library, ldlibrary)
+        elif is_apple_mobile:
+            self.assertEqual(ldlibrary, "Python.framework/Python")
```

Comment:
A nit but this test should be prepared to encounter frameworks with a non-default name, i.e. not `Python`, if the `--with-framework-name` `configure` option is used.

Reply:
It's worth noting that if a custom framework name is used, the testbed won't work out of the box, so testing will at the very least be more complicated. However, assuming whoever is testing that configuration makes the necessary testbed changes, it should be easy enough to account for the custom framework name in the test.